jatszohaz/templates/jatszohaz/profile_detail.html

```diff
@@ -58,5 +59,25 @@ <h3>{% trans 'Rents' %}</h3>
         {% else %}
             {% trans "No rents yet." %}
         {% endif %}
+
+        <h3>{% trans 'Comments' %}</h3>
+        {% for comment in comments %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    {% trans 'User' %}: <a href="{{ comment.user.get_absolute_url }}">{{ comment.user.full_name2 }}</a><br/>

```

Comment:
```suggestion
                    {% trans 'User' %}: <a href="{{ comment.member.get_absolute_url }}">{{ comment.member.full_name2 }}</a><br/>
```

Szerintem itt nem a létrehozót, hanem azt írod ki, akiről szól.